stack-cmbm/recipes/deploy-vagrant.rb

```diff
@@ -0,0 +1,57 @@
+include_recipe 'nginx-app::server'
```

Comment:
Maybe add some fatal chef error on `if is_aws` to ensure this is only run on vagrant?